standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/PropertyServlet.java

```diff
@@ -359,14 +372,34 @@ public static Server startServer(Configuration conf) throws Exception {
     ServletHandler handler = new ServletHandler();
     server.setHandler(handler);
     ServletHolder holder = handler.newServletHolder(Source.EMBEDDED);
-    holder.setServlet(new PropertyServlet(conf)); //
-    handler.addServletWithMapping(holder, "/"+cli+"/*");
+    holder.setServlet(servlet); //
+    handler.addServletWithMapping(holder, "/"+path+"/*");
     server.start();
     if (!server.isStarted()) {
-      LOGGER.error("unable to start property-maps servlet server, path {}, port {}", cli, port);
+      LOGGER.error("unable to start property-maps servlet server, path {}, port {}", path, port);
     } else {
       LOGGER.info("started property-maps servlet server on {}", server.getURI());
     }
     return server;
   }
+
+  /*
+  public static void main(String[] args) throws Exception {
+    HttpServlet servlet = createServlet(null);
+    ServletContextHandler context = new ServletContextHandler(ServletContextHandler.NO_SESSIONS);
+    context.setContextPath("/");
+    ServletHolder servletHolder = new ServletHolder(servlet);
+    servletHolder.setInitParameter("javax.ws.rs.Application", "ServiceListPublic");
+    context.addServlet(servletHolder, "/*");
+    context.setVirtualHosts(null);
+    context.setGzipHandler(new GzipHandler());
+
+    Server httpServer =
+        new Server(PropertyUtil.propertyAsInt(System.getenv(), "REST_PORT", 8181));
```

Comment:
8181 is a pretty generic port. It could cause port conflict.

Reply:
That main() method is entirely in comments it seems.